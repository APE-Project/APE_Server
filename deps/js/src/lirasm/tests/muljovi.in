	ptr = allocp 8

	a = immi 65536
	b = immi 32767
	c = muljovi a b ovf
	sti c ptr 0

	j done

ovf:	i = immi 12345678
	sti i ptr 0

done:	res = ldi ptr 0
	reti res

