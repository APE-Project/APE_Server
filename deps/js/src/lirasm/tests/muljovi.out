Output is: 2147418112
